src/lib/radio/radio.ts

```diff
@@ -117,6 +120,15 @@ export class MdRadioGroup implements AfterContentInit, ControlValueAccessor {
   }
 
   @Input()
+  get required(): boolean {
+    return this._required;
+  }
+
+  set required(value) {
+    this._required = (value != null && value !== false) ? true : null;
```

Comment:
I did, I'm not sure but initially broke the code and the test, then I used the [required] instead of [attr.required] and it worked. is this a correct behaviour I mean [attr.required] not having the same effect as [required]? thanks